src/components/visualizations/utils/test/drilldownEventing.spec.tsx

```diff
@@ -506,4 +506,105 @@ describe("Drilldown Eventing", () => {
             expect(drillConfig.onFiredDrillEvent).toHaveBeenCalled();
         });
     });
+
+    describe("Drilling in Combo chart", () => {
+        const columnPoint: IHighchartsPointObject = {
+            ...point,
+            series: { type: VisualizationTypes.COLUMN },
+        } as any;
+
+        const linePoint: IHighchartsPointObject = {
+            ...point,
+            x: 2,
+            y: 3,
+            series: { type: VisualizationTypes.LINE },
+            drillIntersection: [
+                {
+                    id: "id4",
+                    title: "title4",
+                    header: {
+                        identifier: "identifier4",
+                        uri: "uri4",
+                    },
+                },
+                {
+                    id: "id5",
+                    title: "title5",
+                    header: {
+                        identifier: "identifier5",
+                        uri: "uri5",
+                    },
+                },
+                {
+                    id: "id6",
+                    title: "title6",
+                    header: {
+                        identifier: "identifier6",
+                        uri: "uri6",
+                    },
+                },
+            ],
+        } as any;
+
+        it("should return chart type for each point", () => {
+            const drillConfig: IDrillConfig = { afm, onFiredDrillEvent: jest.fn() };
+            const target: any = { dispatchEvent: jest.fn() };
+            const pointClickEventData: Highcharts.DrilldownEventObject = {
+                point: columnPoint,
+                points: [columnPoint, linePoint],
+            } as any;
+
+            chartClick(drillConfig, pointClickEventData, target as EventTarget, VisualizationTypes.COMBO2);
+
+            jest.runAllTimers();
+
+            const drillContext = target.dispatchEvent.mock.calls[0][0].detail.drillContext;
+
+            expect(drillConfig.onFiredDrillEvent).toHaveBeenCalled();
+            expect(drillContext).toEqual({
+                type: VisualizationTypes.COMBO,
+                element: "label",
+                points: [
+                    {
+                        x: columnPoint.x,
+                        y: columnPoint.y,
+                        intersection: columnPoint.drillIntersection,
+                        type: VisualizationTypes.COLUMN,
```

Comment:
do not use `VisualizationTypes.COLUMN` as you are refering to somethinf else than visualization type. It's series type or something like that. That should have it's own constant

Reply:
Defined new constants `SeriesChartTypes`